qutebrowser/browser/downloads.py

```diff
@@ -180,7 +180,7 @@ def transform_path(path):
     path = utils.expand_windows_drive(path)
     # Drive dependent working directories are not supported, e.g.
     # E:filename is invalid
-    if re.match(r'[A-Z]:[^\\]', path, re.IGNORECASE):
+    if re.fullmatch(r'[A-Z]:[^\\]', path, re.IGNORECASE):
```

Comment:
This should be `re.search` with a `^` anchor added to the regex, as what we want here is really any path starting with something like `E:`.

Reply:
Noted, corrected :)